cubic-server/world-storage/ChunkColumn.hpp

```diff
@@ -0,0 +1,107 @@
+#include <cstdint>
+#include <array>
+#include <vector>
+#include <deque>
+
+#include "Entity.hpp"
+
+#ifndef CHUNKCOLUMN_HPP
+#define CHUNKCOLUMN_HPP
+
+#define CHUNK_2D_SIZE 16*16
+#define CHUNK_3D_SIZE CHUNK_2D_SIZE*16
+#define BIOME_3D_SIZE 4*4*4
+#define NB_OF_CHUNKS 20
+
+struct _3d_pos {
+    int x;
+    int y;
+    int z;
+};
```

Comment:
Is this supposed to be here ?
I think putting it at another place to be usable by the rest of the code would be quite useful